Userland/Libraries/LibWeb/HTML/Scripting/Fetching.cpp

```diff
@@ -280,6 +280,50 @@ static void set_up_module_script_request(Fetch::Infrastructure::Request& request
     request.set_priority(options.fetch_priority);
 }
 
+// https://html.spec.whatwg.org/multipage/webappapis.html#get-the-descendant-script-fetch-options
+WebIDL::ExceptionOr<ScriptFetchOptions> get_descendant_script_fetch_options(ScriptFetchOptions original_options, const URL::URL& url, EnvironmentSettingsObject& settings_object)
```

Comment:
East const should be used for `url`.

Do we need to create pass `original_options` by value, or can we use a const reference